pkg/api/const.go

```diff
@@ -439,7 +439,7 @@ const (
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
 	// DefaultKubernetesClusterSubnetIPv6 specifies the IPv6 default subnet for pods.
-	DefaultKubernetesClusterSubnetIPv6 = "fd00:101::/8"
+	DefaultKubernetesClusterSubnetIPv6 = "fc00::/8"
```

Comment:
This change isn't a doc change (although of course it's good to document it :))

Just to make sure we want to deliver this change?

Reply:
yeah, we want this to be changed.

Reply:
got it! will separate such changes next time and create separate PRs. Thanks @jackfrancis!